### PR TITLE
Update popup for no-solver access

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
Catching up on an old obligation
image

### CODE REVIEW

File: helpers/layout.py [hunk 1]
```diff
@@ -83,12 +83,12 @@ def _dcc_radio(name, suffix):
...
-            href="https://docs.dwavesys.com/docs/latest/doc_ide_user.html",
-            style={"display":"inline-block"}),
-        "documentation"],
-            style={"display":"inline-block"}),])]
+        html.Div(['If you are running in an online IDE, see the ',
+        Link(children=[html.Div('system documentation')],
+            href='https://docs.dwavesys.com/docs/latest/doc_leap_dev_env.html',
+            style={'display':'inline-block'}),
+        ' on supported IDEs.'],
+            style={'display':'inline-block'}),])]
```
Comment (thisac) [inline helpers/layout.py:91]: Why the change to single quotes? I'd say keep double for consistency.

**Review by thisac — APPROVED** [review]

File: helpers/layout.py (hunk 1 above)
Comment (JoelPasvolsky) [reply helpers/layout.py]: Updated, thanks
